packages/frontend/app/lib/utils.ts

```diff
@@ -0,0 +1,6 @@
+import { clsx, type ClassValue } from 'clsx'
+import { twMerge } from 'tailwind-merge'
+
+export function cn(...inputs: ClassValue[]) {
+  return twMerge(clsx(inputs))
+}
```

Comment:
This function is already in generalUtils

Reply:
noted, deleting & fixing import where it is currently used